<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailSendDialog</class>
 <widget class="QDialog" name="EmailSendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>394</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>394</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Отправка поздравлений</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="txtSubject">
       <property name="text">
        <string>Тема:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="senderField">
       <property name="text">
        <string>congratulation.delivery@mail.ru</string>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="txtFrom">
       <property name="text">
        <string>ОТ КОГО:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="txtTo">
       <property name="text">
        <string>email</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="subjectField">
       <property name="text">
        <string>Поздравляю!</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="recipienEmailtField"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="choiceListBirthdayPersons"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>КОМУ:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="congratulationText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="choicePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txtPattern">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вы можете использовать в качестве поздравления один из стандартных шаблонов. А также написать свое поздравление.Не забудьте в конце письма оставить подпись.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>158</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
